Cython/Compiler/ModuleNode.py

```diff
@@ -1553,6 +1554,27 @@ def generate_new_function(self, scope, code, cclass_entry):
         code.putln(
             "}")
 
+    def generate_del_function(self, scope, code):
+        tp_slot = TypeSlots.ConstructorSlot("tp_finalize", '__del__')
```

Comment:
You can use `TypeSlots.get_slot_code_by_name("tp_finalize")` to get the slot code, rather than constructing the `Slot` instance here.